.ci/verify-shellcheck

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env bash
+
+set -e
+set -o pipefail
+
+FILE_LIST=" \
+    ./install_on_demand/.etcdctl \
+    ./install_on_demand/.table \
+    ./install_on_demand/.shrc \
+    ./install_on_demand/.k9s\
+    ./install_on_demand/.wireguard\
+    ./hacks/print-etcd-cheatsheet\
+    ./hacks/ops-pod\
+    ./hacks/install_etcdctl\
+    ./hacks/install_k9s\
+    .ci/test\
+    .ci/build\
+    ./dotfiles/.bashrc\
+    ./dotfiles/.bash_aliases\
+    ./dotfiles/.bash_profile\
+    ./dotfiles/.bash_prompt\
+    ./.ci/verify-shellcheck\
+    ./.ci/verify-bandit\
+"
+
+if ! command shellcheck 1>&2 >/dev/null; then
+    apk add shellcheck
```

Comment:
Can you adapt this and the `verify-bandit` scripts so that they can be executed on mac when developing this locally? 

It probably won't be easy to automatically install these tools in mac, so some info message could be printed to install the `shellcheck`, `bandit` and `shellcheck-sarif` commands manually. e.g.: https://github.com/gardener/gardener/blob/3fbaf73c788721c4bea38cea0b07ee2a53202258/hack/kind-up.sh#L110-L117

Additionally, I think that we should specify the concrete version of `shellcheck` and `bandit ` to use so that we have reproducible executions and we know which set of checks/filters were used to generate the reports.

Reply:
I would expect that users have those configured in their IDEs/text editors.
I am adding a Makefile and directions for the users how to build their container.

I will pin the `bandit` version (as it's easy to do so here), but don't want to pin the `shellcheck` version. I don't think I will be able to pin the `shellcheck` version to something that will work in an universal manner.
At worse somebody will be able to provide (and get approved) a PR that covers some newer rule that is missed in the build process.